js-src/Reader.spec.ts

```diff
@@ -206,7 +204,7 @@ describe("Reader", () => {
       path: "./tests/fixtures/C_with_CAWG_data.jpg",
     });
 
-    // Call postValidateCawg to decode CAWG assertions
+    // this is now a NO-OP
```

Comment:
Why leave it in if it's a no-op? Backwards compatibility?

Reply:
Yes. I just told people they had to use that method.